src/doctopus/web.clj

```diff
@@ -34,16 +35,42 @@
       response
       (four-oh-four (:uri request)))))
 
-(defn serve-index
-  "Just returns our index.html, with the content-type set correctly"
-  [_]
-  (-> (slurp (io/resource "index.html"))
+(defn serve-html
+  "Just returns html, with the content-type set correctly"
+  [template]
+  (-> template
       (ring-response/response)
       (ring-response/content-type "text/html")))
 
+(defn serve-index
+  [_]
+  (serve-html (templates/index)))
+
+(defn serve-add-head-form
+  [_]
+  (serve-html (templates/add-head)))
+
+(defn serve-add-tentacle-form
+  [_]
+  (serve-html (templates/add-tentacle)))
+
+(defn add-head
```

Comment:
Probably eventually gonna want some of this to live in a different NS, just to keep stuff tidy?